[Enhancement] : Title favicon [GSSOC'23]

imohit1o1 opened the pull request:
Fixed this issue #197 

Add the favicon to the title bar

imohit1o1 committed:
Title favicon

add the favicon to the title bar

AshutoshDash1999 commented:
I think this is not working. Please review your code.
![image](https://github.com/Bookingjini-Labs/bookingjini-icons/assets/46455250/a8169b1c-8a1f-4a8b-8f9f-12c58f622d22)

imohit1o1 committed:
Merge branch 'Bookingjini-Labs:dev' into dev

imohit1o1 committed:
Add the favicon to the title bar

I reviewed the code and now its working properly. You can review it now
and merge it

itsmohit097 commented:
I commit again you can check above. It is working properly now.
OR
should i close this PR and open a new PR. Please guide me.

AshutoshDash1999 reviewed (commented):
why have you added 2 icons? also what is `fav`? It doesn't mean anything. Please give good naming.

click on the deployed preview link. The favicon still not showing up.

imohit1o1 committed:
Revert "Merge branch 'Bookingjini-Labs:dev' into dev"

This reverts commit 667a0d5c18d013e68637454f5de6b76e6a4c45d2, reversing
changes made to 177a474df3d319970240cb8dc59e441a91da2c29.
